forge/forge/op/eval/forge/cast.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Union
+import torch
+
+from ..interface import PyEltwiseUnaryOp
+from ....tensor import forge_dataformat_to_pytorch_dtype, pytorch_dtype_to_forge_dataformat
+from forge._C import DataFormat
+
+
+class Cast(PyEltwiseUnaryOp):
```

Comment:
cc @dgolubovicTT This one will be a good addition to testing data formats. 

After the push of ResNet 50, let's checkout how to test out and properly model data formats through TVM and FFE. If we don't do it sooner rather than latter, it'll probably bite us for other models as well